src/pages/ChatPage.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import './ChatPage.css';
+
+const ChatPage = ({location, history}) => {
+    const name = location.state;
+    const [message, setMessage] = useState([]);
+    const userText=['안녕하세요', '샘플 데이터입니다', '반가워요', '날씨가 어때요?', '감사합니다', '좋은하루 되세요', '사랑해요ㅎ.ㅎ'];
+    const [value, setValue] = useState('');
```

Comment:
```suggestion
    const [inputMessage, setInputMessage] = useState('');
```
변수명을 value 라고 이름을 지으시면 어떤 값을 담고 있는지 전혀 유추를 할 수가 없습니다.
자료형까지 말이죠.